packages/manager/cypress/e2e/core/linodes/resize-linode.spec.ts

```diff
@@ -80,10 +78,7 @@ describe('resize linode', () => {
 
       containsVisible('OFFLINE');
 
-      cy.intercept(
-        'POST',
-        apiMatcher(`linode/instances/${linode.id}/resize`)
-      ).as('linodeResize');
+      mockGetFeatureFlagClientstream().as('linodeResize');
```

Comment:
```suggestion
      interceptLinodeResize().as('linodeResize');
```
(This also applies to the replacements later in the file)

Interestingly the tests still run and pass just fine -- it calls into question whether the intercepts are even really necessary in these tests!

Reply:
Haha we've all been there 😅